config/ui-extensions/apigateways/general

```diff
@@ -6,5 +6,9 @@ urlPath: apigateways
 category: Kyma
 name: APIGateway
 scope: cluster
+features:
+  actions:
+    disableCreate: true
```

Comment:
Just for me to understand, does that mean that user needs to create the apigateway CR from e.g. terminal for the first time?

Reply:
it's created by Lifecycle Manager

Reply:
I guess OS is not a problem, as the installation documentation mentions creating CR from CLI anyway